src/components/Checkbox.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import { Typography, TextField, Checkbox } from '@material-ui/core';
+import Container from '@material-ui/core/Container';
+import CheckBoxIcon from '@material-ui/icons/CheckBox';
+
+const useStyles = makeStyles((theme: Theme) =>
```

Comment:
had to switch out the unchecked box icon with another icon (because afaik there isn't a way to change checkbox line width?), but even when i positioned them to be ontop of each other, clicking one would move down other elements. is there another way i could fix that besides absolute positioning?